README.md

```diff
@@ -188,84 +182,53 @@ and install the dependencies:
 cd bandada && yarn
 ```
 
-## 📜 Usage
-
-### Copy the API environment variables
+## 🔧 Configuration
 
-Run this command to copy the [environment variables used by the API](https://github.com/privacy-scaling-explorations/bandada?tab=readme-ov-file#environment-variables):
-
-```bash
-cp apps/api/.env.example apps/api/.env
-```
+### Prerequisities
 
-### Starting applications
+-   [NodeJS](https://nodejs.org/en) >= v18.17.0
+-   [Yarn](https://yarnpkg.com/) >= v3.4.1
```

Comment:
I think this is not necessary because it is downloaded locally with the repo code: https://github.com/privacy-scaling-explorations/bandada/tree/main/.yarn/releases

Reply:
My bad! You are correct!